server/src/test/java/com/broadcom/lsp/cobol/usecases/TestEmptyLinesAllowedBetweenContinuationLines.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package com.broadcom.lsp.cobol.usecases;
+
+import com.broadcom.lsp.cobol.usecases.engine.UseCaseEngine;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+/** Tests that empty lines and comments are allowed between two continuation Lines. */
+class TestEmptyLinesAllowedBetweenContinuationLines {
+  private static final String TEXT =
+      "       Identification Division.\n"
+          + "       Program-id. HELLO-WORLD.\n"
+          + "       Data Division.\n"
+          + "       Working-Storage Section.\n"
+          + "       Procedure Division.\n"
+          + "           DISPLAY 'TEST TEXT CONCATENATED\n"
+          + "           \n"
+          + "      * TEST COMMENTS BETWEEN CONTINUOUS LINES\n"
+          + "      -      'AT DIFFERENT LINE'.";
```

Comment:
Please, add a variable usage below the concatenated lines to make sure the positions for semantic elements don't change

Reply:
Done